.travis.yml

```diff
@@ -43,58 +43,48 @@ services: postgresql
 sudo: required
 
 install:
-  - composer install --working-dir=src
+  - ./utils/prepare-test -afty
+  - composer install --prefer-dist --working-dir=src
   - ./install/scripts/install-spdx-tools.sh
   - ./install/scripts/install-ninka.sh
 
-before_script:
-  - ./utils/prepare-test -afty
-
-env:
-  global:
-    - PHPTESTSUITE=''
-    - CHECKBEFORE='cppcheck -q -isrc/nomos/agent_tests/testdata/NomosTestfiles/ -isrc/testing/dataFiles/ --suppress=*:src/copyright/agent/json.hpp src/'
-    - MAKETARGETS='all test'
-  matrix:
-    - PHPTESTSUITE='Fossology PhpUnit Test Suite' CHECKBEFORE='' MAKETARGETS='build-lib VERSIONFILE build-cli'
-
 script:
-  - set -e
   - src/testing/syntax/syntaxtest.sh
-  - if [[ ${TRAVIS_PHP_VERSION:0:3} == "5.6" ]]; then composer require --no-update phpunit/phpunit ^5; fi
-  - if [[ ${TRAVIS_PHP_VERSION:0:3} == "7.0" ]]; then composer require --dev --no-update phpunit/phpunit ^6; fi
-  - if [[ ${TRAVIS_PHP_VERSION:0:3} == "7.1" ]]; then composer require --dev --no-update phpunit/phpunit ^7; fi
-  - if [[ ! -z "${CHECKBEFORE}" ]]; then ${CHECKBEFORE}; fi
-  - if [[ ! -z "${MAKETARGETS}" ]]; then make ${MAKETARGETS}; fi
-  - if [[ ! -z "${PHPTESTSUITE}" ]]; then src/vendor/bin/phpunit -csrc/phpunit.xml --testsuite="${PHPTESTSUITE}"; fi
-  - set +e
+  - make build-lib VERSIONFILE build-cli
+  - src/vendor/bin/phpunit -csrc/phpunit.xml --testsuite="Fossology PhpUnit Test Suite"
 
 after_script:
```

Comment:
When is this part ran? every step in the matrix either has `after_script: skip` or `<<: *compiler-tests` which implies `after_script: skip`?

Reply:
The `after_script` is only executed in the matrix expansion. Actually three jobs (php 7.0, 5.6 and 7.1).

See: https://travis-ci.org/fossology/fossology/jobs/421474343

Reply:
It is done in the PHPUnit tests?

Reply:
Yes, check the end of the [travis-ci log](https://travis-ci.org/fossology/fossology/jobs/421474343).